osquery/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(OSQUERY_BENCHMARKS "")
 set(OSQUERY_KERNEL_BENCHMARKS "")
 
 # The 'additional' source files are NOT included with SDK (libosquery_additional).
-set(OSQUERY_ADDITIONAL_SOURCES "")
+set(OSQUERY_ADDITIONAL_SOURCES main/empty.cpp)
```

Comment:
This is not a good solution. Can you describe the problem you're having with `libosquery_additional`?

Reply:
It was not linked at all by Xcode…